### PR TITLE
app-text/master-pdf-editor: version bump to 5.1.30 and 5.1.36.

### PR DESCRIPTION
**Pull request by jpataias**
Removed 4.3.82, 5.0.36 and 5.1.00 as sources are no longer available.
Version bump to 5.1.30 and 5.1.36.

### CODE REVIEW

File: app-text/master-pdf-editor/master-pdf-editor-5.1.36.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+S="${WORKDIR}/${PN}-${PV%%.*}"
+
+src_install() {
+	insinto /opt/${PN}
+	doins -r fonts lang stamps templates masterpdfeditor5.png
+
+	exeinto /opt/${PN}
+	doexe masterpdfeditor5
+
+	dosym ../${PN}/masterpdfeditor5 /opt/bin/masterpdfeditor5
```
Comment (titanofold) [inline app-text/master-pdf-editor/master-pdf-editor-5.1.36.ebuild]: Style nitpick: I would move this line up one to keep it next to the thing it points to. Basically, keep related things together. (_In this paragraph, I'm talking about the executable...._)

Comment (jpataias) [reply app-text/master-pdf-editor/master-pdf-editor-5.1.36.ebuild]: Thank you for your feedback. I made the suggested changes.

**Comment by jpataias** [comment]
Damn, shame on me. All fixed now.

**Comment by mgorny** [comment]
Thanks. I'll merge now.

**Pull request closed without merging**